time-series/time-series-api/src/main/java/com/powsybl/timeseries/TimeSeries.java

```diff
@@ -406,7 +438,8 @@ static void checkCsvHeader(TimeSeriesCsvConfig timeSeriesCsvConfig, String[] tok
         }
     }
 
-    static Map<Integer, List<TimeSeries>> parseCsv(BufferedReader reader, TimeSeriesCsvConfig timeSeriesCsvConfig) {
+    static Map<Integer, List<TimeSeries>> parseCsv(BufferedReader reader, TimeSeriesCsvConfig timeSeriesCsvConfig,
```

Comment:
perhaps we could also use this PR to change all TimeSeries with omitted type parameters (sonar indicates to provide the parametrized type)
concerned functions are at least : parseCsv, createTimeSeries, readCasValues, writeJson, toJson, parseChunks, parseJson

Reply:
It would be better to create another PR for this since it's a different issue (but it might be a good idea)